packages/volto/src/components/theme/RouteAnnouncer/RouteAnnouncer.jsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+
+function RouteAnnouncer() {
+  const [pageTitle, setPageTitle] = useState('');
+
+  function updatePage(title) {
+    setPageTitle(title);
+    document.activeElement.blur();
+  }
+
+  useEffect(() => {
+    function handlePop(event) {
+      const pageTitle = event.target.document.title;
+      updatePage(pageTitle);
+    }
+
+    const observer = new MutationObserver((mutationList) => {
+      for (const mutation of mutationList) {
+        if (mutation.type === 'childList') {
+          for (const node of mutation.addedNodes) {
+            if (node.nodeType === Node.TEXT_NODE && node.textContent) {
+              updatePage(node.textContent);
+            }
+          }
+        }
+      }
+    });
+    observer.observe(document.querySelector('title'), {
+      characterData: true,
+      attributes: true,
+      childList: true,
+      subtree: true,
+    });
+    window.addEventListener('popstate', handlePop);
+
+    return () => {
+      observer.disconnect();
+      window.removeEventListener('popstate', handlePop);
+    };
+  }, []);
```

Comment:
By just looking at the code I would say that eslint should complain that this array should at least contain the `updatePage` function. Would it be possible to move it inside the `useEffect` next to `handlePop`?